bsp/nrf5x/nrf52840/board/fal_cfg.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * File      : fal_cfg.h
+ * This file is part of FAL (Flash Abstraction Layer) package
+ * COPYRIGHT (C) 2006 - 2018, RT-Thread Development Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
```

Comment:
Please use correct copyright file header. For these files, only Apach License v2.0 codes are acceptable.